core/src/transport.rs

```diff
@@ -35,11 +35,11 @@ use std::{
 };
 
 pub mod and_then;
-pub mod choice;
 pub mod dummy;
 pub mod map;
 pub mod map_err;
 pub mod memory;
+pub mod or;
```

Comment:
This is a breaking change. I'd like to have this non-breaking please. Should be easy with another module and type aliases.